Editor/Editors/CinemachineSplineDollyEditor.cs

```diff
@@ -8,8 +9,51 @@ namespace Cinemachine.Editor
     [CanEditMultipleObjects]
     sealed class CinemachineSplineDollyEditor : BaseEditor<CinemachineSplineDolly>
```

Comment:
Since you are excluding and custom-implementing all the fields, it might make sense to not inherit BaseEditor.  Are you using any of its features, or is it just an encumbrance?

Reply:
I use a few things:
- Spline is not excluded, I draw that with BaseEditor. 
- BeginInspector to call Update serializedObject.
- serializedObject

Reply:
You are right. I don't need it. I'll use normal editor